org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ProjectUtils.java

```diff
@@ -416,12 +416,12 @@ public static IPath resolveGlobPath(IPath base, String glob) {
 				}
 				prefixLength += 1;
 			}
-			Path prefix = pattern.uptoSegment(prefixLength).toFile().toPath();
-			Path remain = pattern.removeFirstSegments(prefixLength).toFile().toPath();
+			IPath prefix = pattern.uptoSegment(prefixLength);
+			IPath remain = pattern.removeFirstSegments(prefixLength).setDevice("");
```

Comment:
In Windows, `removeFirstSegments` won't remove the device name (e.g. `C:`), so setting an empty device is used. `pattern.makeRelativeTo(prefix)` can also do the same job, but I keep `removeFirstSegments` for minorly better performance.